<launch>
	<!-- Max's trajectory controller -->
	<node name="mico_joint_velocity_controller" pkg="moveit_utils" type="mico_joint_velocity_controller" />
	
	<!-- Mico services: These services interface with moveit and the controller -->
	<node name="mico_moveit_joint_pose_service" pkg="moveit_utils" type="mico_moveit_joint_pose_service" />
	<node name="mico_moveit_caresianpose_service" pkg="moveit_utils" type="mico_moveit_cartesianpose_service" />
	<node name="mico_moveit_caresianpose_obstacle_service" pkg="moveit_utils" type="mico_moveit_cartesianpose_obstacle_service" />
	
	<!-- safety service -->
	<node name="mico_nav_safety_service" pkg="moveit_utils" type="mico_nav_safety_service" output="screen"/> 
	
	<!-- Jivko's angular controler -->
	<node name="angular_vel_control_srv" pkg="moveit_utils" type="angular_vel_control_srv" />
	
	 <!-- Launch moveit & rviz -->
    <include file="$(find mico_moveit_fullsegbot_config)/launch/mico_moveit_bringup.launch" />

</launch>
